src/vtpty/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ if(LIBSSH2_FOUND)
     target_include_directories(vtpty PRIVATE ${LIBSSH2_INCLUDE_DIRS})
     target_link_libraries(vtpty PRIVATE ${LIBSSH2_LINK_LIBRARIES})
 elseif(WIN32)
-    target_link_libraries(vtpty PRIVATE $<IF:$<TARGET_EXISTS:Libssh2::libssh2_shared>,Libssh2::libssh2_shared,Libssh2::libssh2_static>)
+    target_link_libraries(vtpty PRIVATE Libssh2::libssh2)
```

Comment:
This is weird, because i've got it to run. `master` branch should be already working. I don't mind this change regardless, because we should not link against the shared version but only against the static one. :)